src/line.rs

```diff
@@ -969,6 +969,10 @@ fn parse_time_type(c: &str) -> Option<TimeType> {
 }
 
 impl LineParser {
+    pub fn new() -> Self {
+        Self::default()
+    }
```

Comment:
I'd prefer to just leave this out.

Reply:
Okay, maybe add a `#[deprecated]` at least?

Reply:
Done. I still prefer `new()` myself.

Reply:
Why?

Reply:
It is shorter, familiar, and still somewhat of a convention to have `new`. Of course it is completely redundant if you have `default()`.

Reply:
Fair -- my counter arguments would be:

* Short: 3 vs 7 characters, not an important difference
* Familiar: here, too, the difference seems very small
* Convention: both are a convention, but there is more type system pressure for `Default`

In the end, given that it is redundant, I prefer having less code.